aix/generate_wazuh_packages.sh

```diff
@@ -40,12 +40,21 @@ show_help() {
   echo "    -e,  --environment            Install all the packages necessaries to build the RPM package"
   echo "    -s,  --store  <path>          Directory to store the resulting RPM package. By default: ${target_dir}"
   echo "    -p,  --install-path <path>    Installation path for the package. By default: ${install_path}"
-  echo "    -c,  --checksum <path>        Compute the SHA512 checksum of the RPM package."
+  echo "    -c,  --checksum <path>        Compute the SHA512 checksum of the RPM package. OpenSSL is required."
   echo "    -h,  --help                   Shows this help"
   echo
   exit $1
 }
 
+check_openssl() {
+  if [[ -z "$(command -v openssl)" ]] && [[ "${compute_checksums}" = "yes" ]]; then
+    echo "OpenSSL is not installed. OpenSSL is required to get the package checksum."
+    return 1
```

Comment:
```suggestion
    help 1
```

Reply:
This error is not related to an error in the parameters but to specific information of a package that is missing when using that parameter